rllib/algorithms/ppo/tf/ppo_tf_rl_module.py

```diff
@@ -77,10 +60,9 @@ def output_specs_train(self) -> List[str]:
 
     @override(TfRLModule)
     def _forward_train(self, batch: NestedDict):
-        obs = batch[SampleBatch.OBS]
-        encoder_out = self.shared_encoder(obs)
-        action_logits = self.pi(encoder_out)
-        vf = self.vf(encoder_out)
+        encoder_out = self.encoder(batch)
```

Comment:
Can you make sure we take care of STATE_OUT here as well? similar to torch? TF and torch should be maximally consistent going fwd.

Reply:
Done! 🙂